### PR TITLE
feat: notebook about webhooks

### PR DESCRIPTION
**Pull request by p-desaintchamas**

### CODE REVIEW

File: recipes/webhooks_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+    "\n",
+    "```python\n",
+    "# file main.py\n",
+    "from fastapi import FastAPI\n",
+    "from .plugin import PluginHandler\n",
+    "\n",
+    "app = FastAPI()\n",
+    "kili = Kili()\n",
+    "\n",
+    "@app.get(\"/\")\n",
```
Comment (gheorghetutunaru) [inline recipes/webhooks_example.ipynb:258]: We're using POST, does get still receive the requests ?

File: recipes/webhooks_example.ipynb [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+    "def main(raw_payload):\n",
+    "    event_type = raw_payload.get('eventType')\n",
+    "    project_id = raw_payload.get('logPayload').get('projectId')\n",
+    "\n",
+    "    if (not project_id):\n",
+    "        print('Invalid projectId')\n",
+    "        return\n",
+    "    \n",
+    "    project_id = payload.get('projectId')\n",
```
Comment (gheorghetutunaru) [inline recipes/webhooks_example.ipynb:267]: Duplicate line

File: recipes/webhooks_example.ipynb [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+    "    plugin = PluginHandler(kili, project_id)\n",
+    "\n",
+    "    if (not event_type):\n",
+    "        print('Invalid event')\n",
+    "        return\n",
+    "\n",
+    "    payload = raw_payload.get('payload')\n",
+    "    label = payload.get('label')\n",
+    "    asset_id = payload.get('assetId')\n",
```
Comment (gheorghetutunaru) [inline recipes/webhooks_example.ipynb:276]: Should be asset_id, no ?

Comment (p-desaintchamas) [reply recipes/webhooks_example.ipynb:276]: Nope, we have camel case in the event's payload

Comment (gheorghetutunaru) [reply recipes/webhooks_example.ipynb:275]: It's camelCase in the raw_payload, but snake case for the `payload` :

image

Comment (gheorghetutunaru) [reply recipes/webhooks_example.ipynb:275]: Not really consistent, but it was done to automatically call the good handler for the plugins

File: src/kili/mutations/plugins/__init__.py [hunk 4]
```diff
@@ -69,10 +69,19 @@ def create_webhook(
...
         Args:
-            webhook_url: URL receiving post requests on events on Kili
+            webhook_url: URL receiving post requests on events on Kili. The payload will be the following:
+             - eventType: the type of event called
+             - logPayload:
+               - runId: a unique identifier of the run for observability
+               - projectId: the Kili project the webhook is called on
+             - payload: the event produced, for example for `onSubmit` event:
+              - label: the label produced
+              - assetId: the asset on which the label is produced
```
Comment (gheorghetutunaru) [inline src/kili/mutations/plugins/__init__.py:84]: also `asset_id`

**Review by gheorghetutunaru — APPROVED** [review]

**Review by florianlega — APPROVED** [review]

**Pull request merged**